ci/do_ci.sh

```diff
@@ -85,11 +85,20 @@ CI_TARGET=$1
 
 if [[ $# -gt 1 ]]; then
   shift
+  # If this is a fuzz test, add _with_libfuzzer
+  if [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
+    FUZZ_TEST_TARGETS=$*"_with_libfuzzer"
+  fi
   TEST_TARGETS=$*
 else
   TEST_TARGETS=//test/...
+  if [[ "$CI_TARGET" == "bazel.fuzz" ]]; then
```

Comment:
I think a more robust approach here would be to use tags introduced by `envoy_cc_fuzz_test`. That would allow either a `bazel query` or just a tag filter on the `bazel test`.

Arguably, I should have also done this instead of https://github.com/google/oss-fuzz/blob/master/projects/envoy/build.sh#L21, but we can switch to tags here and maybe do it in `build.sh` some day as a cleanup (no real urgency).

Reply:
+1 `bazel query`

Reply:
I added a tag specifically to the libfuzzer fuzz tests to differentiate them. I could have also done a bazel query on the name "with_libfuzzer"